crates/pkgcraft/src/repo/ebuild/cache/md5_dict.rs

```diff
@@ -303,7 +302,7 @@ impl Cache for Md5Dict {
         // Remove outdated, invalid, and unrelated files as well as their parent
         // directories if empty.
         entries
-            .into_par_iter()
+            .into_iter()
```

Comment:
I'm not sure what is achieved by de-parallelizing file removal which is inherently I/O based and probably is highly affected when run against commits performing sweeping, tree-wide ebuild removals that probably occur semi-regularly for software suites like KDE or GNOME.

Reply:
To have a clean baseline.

Reply:
I'll readd it in a bit.